static/js/stream_settings_ui.js

```diff
@@ -428,6 +428,22 @@ export function render_left_panel_superset() {
     ui.get_content_element($("#manage_streams_container .streams-list")).html(html);
 }
 
+// params {isEmpty: Booliean}
+export function render_on_empty_streams(params) {
+    if (params.is_empty) {
+        $(".nothing-to-show").show();
+        if (is_subscribed_stream_tab_active()) {
+            $(".view_all_stream_link_container").show();
+            $(".create_stream_link_container").hide();
+        } else {
+            $(".view_all_stream_link_container").hide();
+            $(".create_stream_link_container").show();
+        }
+    } else {
+        $(".nothing-to-show").hide();
```

Comment:
We can return early by checking `!is_empty` case and avoid else block. I guess that would be better for readability.